app/locations/service.py

```diff
@@ -0,0 +1,89 @@
+from flask import jsonify
+from typing import List, Dict
+from .poi_suggestion import POI_suggestion
+from .poi import POI
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from app import common
+from datetime import datetime, timezone
+from app.firebase import firestore_db, POI_COLLECTION, POI_PROPOSAL_COLLECTION
+
+poi_collection = firestore_db().collection(POI_COLLECTION)
+poi_proposal_collection = firestore_db().collection(POI_PROPOSAL_COLLECTION)
+
+
+def list_POI() -> List[POI]:
+    """
+    Returns a list of POI objects from the POI collection in Firestore.
+    """
+    return [POI.from_dict(doc.to_dict()) for doc in poi_collection.stream()]
+
+
+def get_details_for_POI(poi_id: str) -> POI:
+    """
+    Returns a POI object based on a given POI ID from the POI collction in Firestore.
+
+    :param poi_id: ID of the POI

```

Comment:
If your function raises any exceptions please document it here so whoever is using it is aware of it